sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -424,7 +424,7 @@ AZ_NODISCARD az_result az_iot_hub_client_methods_parse_received_topic(
  * @param[in] client The #az_iot_hub_client to use for this call.
  * @param[in] request_id The request id. Must match a received #az_iot_hub_client_method_request
  *                       request_id.
- * @param[in] status The status. Must be 200 for "OK", 404 for "error", or 504 for "timeout"
+ * @param[in] status The status.

```

Comment:
Sugg: A code that indicates the result of the method, as defined by the user.

Reply:
updated